docs/index.asciidoc

```diff
@@ -3,6 +3,7 @@
 
 :version:        6.0.0-alpha1
 :major-version:  6.x
+:docker-image:   docker.elastic.co/kibana/kibana:6.0.0-alpha1
```

Comment:
I vaguely recall a mention that nested references for vars aren't recommended, but my tests were successful, so is there a reason not to have `docker.elastic.co/kibana/kibana:{version}`?